internal/uploads/profilePicture.go

```diff
@@ -0,0 +1,21 @@
+package uploads
+
+type ProfileImageUploadRepository interface {
+	GetUploadUrl(ext string, userID string) (string, error)
+}
+
+type Service interface {
+	GetUploadUrl(ext string, userID string) (string, error)
+}
+
+type image struct {
+	imageUploadRepository ProfileImageUploadRepository
+}
+
+func NewProfileImageUploadService(profileImageUploadRepo ProfileImageUploadRepository) Service {
+	return &image{imageUploadRepository: profileImageUploadRepo}
+}
+
+func (i *image) GetUploadUrl(ext string, userID string) (string, error) {
```

Comment:
always get context as the first arg

Reply:
@Sobowale-Olayiwola this still needs to be addressed